### PR TITLE
Add autoupdate config in froala-editor

### PR DESCRIPTION
**Pull request by stefanneculai**
Hello,

I have added autoupdate in package.json and fixed the Froala Editor links and licenses.

### CODE REVIEW

File: ajax/libs/froala-editor/package.json [hunk 1]
```diff
@@ -1,59 +1,38 @@
...
-  "maintainers": [
-    {
-      "name": "Froala"
-    }
+    "rich",
+    "editor"
   ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git@github.com:froala/wysiwyg-editor.git",
```
Comment (PeterDaveHello) [inline ajax/libs/froala-editor/package.json]: should use git protocol instead of ssh here.

Comment (stefanneculai) [inline ajax/libs/froala-editor/package.json]: @PeterDaveHello, right. I have updated it now.

**Comment by PeterDaveHello** [comment]
Please squash them into only one commit, thanks!

**Comment by stefanneculai** [comment]
Unfortunately, I cannot squash them because I made the changes from the online interface because if I clone it locally and then I get into that problem with the Sortable library and unable to commit. Shall I close this one and make a new PR with a single commit?

**Comment by PeterDaveHello** [comment]
New PR will be okay, you can also fetch this PR to local and them squash them, depends on you :smile:

**Comment by stefanneculai** [comment]
Would be great if you could make something with that uppercase problem from the Sortable library.

**Comment by PeterDaveHello** [comment]
Sparse checkout will help you @stefanneculai 
https://github.com/cdnjs/cdnjs/blob/master/sparseCheckout.md

**Comment by stefanneculai** [comment]
Probably you should mention about it in the main Readme so that it's easier to find. :+1:

**Comment by PeterDaveHello** [comment]
@stefanneculai yeah you are right, I am to busy so that many things were just throw on the todo list. I'll be also great if you can send a pr too add it 😃

**Pull request closed without merging**